src/main/scala/wdl4s/expression/FileEvaluator.scala

```diff
@@ -115,7 +115,7 @@ case class FileEvaluator(valueEvaluator: ValueEvaluator, coerceTo: WdlType = Wdl
       case a: Ast if a.isArrayOrMapLookup =>
         evalValue(a) match {
           case Success(f: WdlFile) => Success(Seq(f))
-          case _ => evaluateRecursive(a.getAttribute("rhs"))
+          case _ => evaluateRecursive(a.getAttribute("lhs"))
```

Comment:
Based on the comment [above](https://github.com/broadinstitute/wdl4s/blob/e040cf832588aea71fd5e4eaa0e05a4b397c4bd2/src/main/scala/wdl4s/expression/FileEvaluator.scala#L72-L87), I believe this method only supposed to be used during early evaluation, _before_ a task actually runs.

So `glob("globFile.txt")[count - 1]` should return a `Seq(WdlGlobFile("globFile.txt"))` for this method / case. Right?